src/feature/90-core-dump/feature.py

```diff
@@ -8,15 +8,31 @@
 #include <unistd.h>
 #include <fcntl.h>
 #include <sys/stat.h>
+
+#define REGISTER_STRLEN (WORD_BYTES * 2)
+
+static char hex[] = "0123456789abcdef";
+
+static void register_to_str(mit_WORD reg, char *s)
```

Comment:
Document that s points to a char[REGISTER_STRLEN].

Reply:
Even better, make the argument `char s[REGISTER_STRLEN]`. (I thought that wouldn't work when we were discussing it, but I was wrong: arrays are always pointers in C.)